src/package/nuspec/Microsoft.TestPlatform.CLI.csproj

```diff
@@ -8,5 +8,7 @@
     <!-- Don't include the output.dll and output.runtimeconfig.json files in nuget package -->
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
```

Comment:
Why is this?

Reply:
We're not building this project, it is only required for `dotnet pack`, hence disabled all these options.